tests/pageobjects/Filmstrip.ts

```diff
@@ -210,4 +210,12 @@ export default class Filmstrip extends BasePageObject {
             }
         );
     }
+
+    /**
+     * Counts the displayed remote video thumbnails.
+     */
+    async countVisibleThumbnails() {
```

Comment:
Do we use this function?

I believe we don't need the async/await part here too.

Reply:
Yep, it is used in lobby tests. expect(await p3.getFilmstrip().countVisibleThumbnails()).toBe(3);

Reply:
well we cannot use the await here and there is async isDisplayed executed on filter.

Reply:
Oh my bad GH was not showing the file....

Yeah and the await seems correct too.

Ignore!